extra-web/telegram-desktop/spec

```diff
@@ -1,5 +1,5 @@
-VER=2.4.4
+VER=2.4.6
 SRCS="tbl::https://github.com/telegramdesktop/tdesktop/releases/download/v$VER/tdesktop-$VER-full.tar.gz \
-      git::rename=tg_owt;commit=c73a4718cbff7048373a63db32068482e5fd11ef::https://github.com/desktop-app/tg_owt"
```

Comment:
How is this commit determined and tracked? Couldn't this be addressed with Git submodule?